docs/layouts/shortcodes/generated/artifact_fetch_configuration.html

```diff
@@ -0,0 +1,36 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>user.artifacts.artifact-list</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>List&lt;String&gt;</td>
+            <td>A semicolon-separated list of the additional artifacts to fetch for the job before setting up the application cluster. All given elements have to be valid URIs. Example: s3://sandbox-bucket/format.jar;http://sandbox-server:1234/udf.jar</td>
```

Comment:
Why are we switching to semicolon seperator from commas as used by `--jars`?

Reply:
This is the behavior of the [`ConfigurationUtils`](https://github.com/apache/flink/blob/2cf213ec9e4db81815a9b37013904aa47a94aa01/flink-core/src/main/java/org/apache/flink/configuration/ConfigurationUtils.java#L439), I wrongly assumed before that the separator is a comma, but it has to be a semicolon. Any other `List<String>` typed `ConfigOption` works the same, e.g. the ones in [`CoreOptions`](https://github.com/apache/flink/blob/2cf213ec9e4db81815a9b37013904aa47a94aa01/flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java#L107).

Reply:
Thanks for the clarification. 😐